### DISCUSSION TITLE
Browser Before Send Callback

### DISCUSSION THREAD

**Discussion by johanmolen** [stayallive/wp-sentry#148] [Q&A] [answered]
Hi all!

Does somebody have a good example for a `setBeforeSendCallback` for the browser / javascript side?
I want to ignore some plugins from reporting errors on the browser side.

Thanks in advance!

**Comment by stayallive** [comment, marked as answer]
It's a bit tricky :) But this might help: 

- https://github.com/stayallive/wp-sentry#client-side-hook
- https://docs.sentry.io/platforms/javascript/configuration/filtering/

Something like this could work (I did not test this!):

```php
add_action( 'wp_enqueue_scripts', function () {
	wp_add_inline_script( 'wp-sentry-browser', 'function wp_sentry_hook(options) { options.beforeSend = function (event) { if (event.someProperty) { return null; } return event; } }', 'before' );
} );
```

**Comment by johanmolen** [comment]
@stayallive Awesome! Gonna give it a shot, thanks!
